README.md

```diff
@@ -95,6 +95,20 @@ These resources will be critical to your success in configuring and scaling appl
 + [NVEncode Homepage](https://developer.nvidia.com/nvidia-video-codec-sdk)
 + [NvPipe original source](https://github.com/NVIDIA/NvPipe)
 
+## Contributing
+
+### Code of Conduct
```

Comment:
nit: does it make sense to have this information here at all, instead of just linking to `Contributing.md` ?

Reply:
Agree - but I'm going to keep the Code of Conduct in both places.